presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -98,6 +101,11 @@ Property Name                                                 Description
 
 ``cassandra.split-size``                                      Number of keys per split when querying Cassandra.
 
+``cassandra.splits-per-node``                                 Allows to override the number of splits per node. Only use this
```

Comment:
Suggest: Number of splits per node. By default, the values from the ``system.size_estimates`` table are used. Only override when connecting to Cassandra versions < 2.1.5, which lack the ``system.size_estimates`` table.